www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -17,3 +17,16 @@
  * For more information : contact@centreon.com
```

Comment:
shouldn't the script be for the 19.10.2 ?
how will you cherry-pick these modifications on the 2.8 / 18.10 and 19.04 ?

Reply:
master is the new 20.04... resolving this one